### DISCUSSION TITLE
Accelerate upscaling?

### DISCUSSION THREAD

**Discussion by ConteXCrown** [bloc97/Anime4K#177] [Ideas]
I did a little research on AI recently. Turns out our current hardware its not optimzed for AI yet.
So I found products like a USB Accelerator from coral.ai wich claim to speed up ML processes and I wondered if you could use this with video upscalers like Anime4k.
I think this would be a gamechanger because many people dont want to watch upscaled anime on their desk setups rather on a tablet, laptop, ... and chill on their bed or on the train. This is currently not possible because mobile hardware is not that powerful/optimized and would use a lot of power.
So I wanted to ask if this would be technically possible since I didnt find anymore information on the web.
Thanks for response
![coreal](https://user-images.githubusercontent.com/98663891/171308973-0c423cfc-1760-4d56-b3f3-1edc4cb3e49b.PNG)

**Comment by bloc97** [comment]
This can't be done until libplacebo (the underlying rendering library) supports external library calls. This feature is still being worked on https://code.videolan.org/videolan/libplacebo/-/issues/174 .
